module5/lessons/Week4/CoverLetters.md

```diff
@@ -0,0 +1,189 @@
+
+AWARDS  
+Department of Defense Freedom Award
+ESGR Employer Support Award
+Department of Labor HIRE Vets Medallion Program Platinum Award 2018, 2019, 2020, 2021, and 2022
+
+-------------------------------------------------------------------------------
+
+#### Job Title: Software Engineer I
+**Full Time, Hybrid with 2 days WFH after initial onboarding**  
+Pay: $75,000.00 - $85,000.00 per year  
+
+**Job Description:**  
+As a member of the software engineering team you will work on developing software solutions that best meet the needs of our customers. Under the mentorship of more senior engineers you will participate in all parts of the software development lifecycle. Someone with an interest in building web and mobile applications, the ability to solve problems and provide effective solutions for customers, and a strong desire to learn new programming languages, frameworks and technologies will be the best fit for this position.
+
+**Essential Duties:**  
+With guidance from your mentor you will be expected to.
+* Gather requirements and plan work related to a task.
+* Determine appropriate software design.
+* Implement bug fixes, improvements or new features.
+* Test changes before deployment to production environments.
+* Prepare changes for deployment to production environments.
+
+**Education and Experience**:  
+Required: 
+* Bachelor’s degree from accredited four-year university in Computer Science, Software Engineering or a related field or equivalent experience.
+* 0-2 years of related experience.  
+
+Preferred:   
+* Honorably discharged service veterans.
+
+**Knowledge, Skills, & Abilities:**
+* Self-motivated and a great teammate.
+* Strong communication and organizational skills.
+* Interest in developing web and mobile applications.
+* Proficiency with a high level programming language such as Python, JavaScript, Go, C#, or Java
+* Proficiency with web markup and styling, HTML and CSS.
+* Proficiency with SQL.
+
+</section>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/lW0JbWwrj_0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### First paragraph: The hook 
+
+* Show why you’re interested in the position
+* Mention how you found out about the opening 
+* State the position you’re applying
+* Use the title from the job listing
+
+<section class='call-to-action' markdown='1'>
+
+Write a first paragraph for a cover letter for the job post above.  Be ready to share you paragraph in small groups!
+
+</section>
+
+<section class='instructor-notes' markdown='1'>
+
+Give students a few minutes to draft a paragraph, then put them in breakout rooms to share and discuss.  Go to different rooms and pull out examples of what we like to see, and identify some growth areas to share out with the whole class.
```

Comment:
I like this structure a lot of drafting individually, sharing in small groups, then discussing as a group.